dev-docs/bidders/consumable-server.md

```diff
@@ -0,0 +1,373 @@
+
+<a id="call-Consumable-from-ctv-long-form-video-environment"></a>
+
+### Call Consumable from CTV/long-form video environment
+
+**Before you begin:** Contact your Consumable Exchange Representative to get the `placementId`. You must provide this placement ID to your Prebid Server host company.
+
+**Note:** To implement CTV and long-form video using Prebid Server, follow Prebid's [Long Form Video](/prebid-server/use-cases/pbs-lfv.html) documentation.
+
+To add Consumable as a bidder:
+
+1. Inform your Prebid Server hosting company to add `consumable`as a bidder in the configuration and include the `placementId` that Consumable provides to you at the time of integration.
+2. Define the Consumable-specific parameters at the bidder level. For information about these parameters, see the [Bid request parameters](#bid-request-parameters) section below.
+3. Include any ad unit level required or optional parameters provided in Prebid's [/openrtb2/video](/prebid-server/endpoints/openrtb2/pbs-endpoint-video.html) documentation.
+
+<a id="bid-request-parameters"></a>
+
+## Bid request parameters
+
+For a list of the OpenRTB fields that Consumable supports in bid requests, see [List of supported OpenRTB bid request fields for sellers](https://kb.Consumableexchange.com/publishers/openrtb_integration/list_of_supported_openrtb_bid_request_fields_for_sellers.htm#List_of_supported_OpenRTB_bid_request_fields_for_sellers). The following are the required fields for the various supported media types.
+### Site (Banner, Video, Audio)
+<a id="site"></a>
+You must include these parameters at the bidder level.
+
+{: .table .table-bordered .table-striped }
+| Name        | Scope    | Description                    | Example | Type      |
+|-------------|----------|--------------------------------|---------|-----------|
+| `siteId`    | required | The site ID from Consumable.    | `12345` | `integer` |
+| `networkId` | required | The network ID from Consumable. | `9969`  | `integer` |
+| `unitId` | required | The unit ID from Consumable. | `987654`  | `integer` |
+| `unitName` | required | The unit name from Consumable. | `cnsmbl-unit`  | `string` |
+### App or CTV (Banner, Video, Audio)
+<a id="app"></a>
+You must include these parameters at the bidder level.
+
+{: .table .table-bordered .table-striped }
+| Key | Scope | Type | Description |
+|---|---|---|---|
+| `placementId` | Required | String | An Consumable-specific identifier that is associated with this ad unit. It will be associated with the single size, if the size is provided. This is similar to a placement ID or an ad unit ID that some other modules have. For example, `'0421008445828ceb46f496700a5fa65e'`|
+
+<a id="examples"></a>
```

Comment:
```suggestion
```

All headings can be targeted and thus no additional, empty anchor tags are needed